src/Select/web/Select.js

```diff
@@ -248,4 +261,4 @@ Select.contextTypes = {
   formik: PropTypes.object,
 };
 
-export default Select;
+export default withTheme(Select);
```

Comment:
If everything above is followed, you can go ahead and remove this, since it'll no longer be required.